custom_components/entsoe/api_client.py

```diff
@@ -125,10 +129,12 @@ def parse_price_document(self, document: str) -> str:
                     )
                     continue
 
-                if resolution == "PT60M":
-                    series.update(self.process_PT60M_points(period, start_time))
-                else:
-                    series.update(self.process_PT15M_points(period, start_time))
+                interval = 15 if resolution == "PT15M" else 60
```

Comment:
Do we really need to be explicity here? Can't we just take the resolution and fill that in as the interal, regardless if it is 60, 15, or maybe in the future 5 or 1?

Reply:
Good point, when doing this I found out actually quite a lot of "1 hour assumptions" had been made in this integration. I reworked it now such that it can support any kind of interval as long as it stays within minutes